design/idempotency_ids.md

```diff
@@ -76,13 +85,29 @@ If a transaction learns that it has been in-flight so long that its idempotency
 
 # Considerations
 
-- Additional storage space on the cluster. This can be controlled directly via an idempotency id target bytes knob/config.
+- Additional storage space on the cluster.
 - Potential write hot spot.
```

Comment:
Could you clarify what might cause the write hot spot?

Reply:
Done